setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
     "treelib>=1.5.5",
     "inflect>=2.1.0",
     "humanize>=0.5.1",
-    "ruamel.yaml>=0.15.91",
+    "ruamel.yaml<0.16",
```

Comment:
Just to be safe, let's do
```suggestion
    "ruamel.yaml>=0.15.91,<0.16",
```

Reply:
good point!